docs/proposals/project-user-resource-quotas.md

```diff
@@ -0,0 +1,209 @@
+# Project resource quotas
+
+**Authors**: Lovro Sviben (@lsviben)
+
+**Status**: Draft proposal.
+
+**Issue**: https://github.com/kubermatic/kubermatic/issues/8042
+
+## Goals
+The idea is to allow admins to control how many resources are projects using by introducing resource quotas.
+
+Resources that will be under quota:
+- vCPU
+- RAM
+- storage (node disk size) - TBD, we can remove this as it's not a strict requirement
+
+## Non-Goals
+- The quotas won't work for imported external clusters.
+- The quotas won't restrict the usage of resources for the control plane on the seed cluster.
+- Nodes which join the cluster using other means then through KKP are not supported in the quotas.
+
+## Motivation and Background
+
+Clients would like to have options to fine-grain restrict resource usage per project.
```

Comment:
Please use "KKP Admins", so we do not always talk in us+customer terms, but us+community. I do not want to give off the vibe that what we do here is only to satisfy customers :)